extensions/linkedin/0.1.0/schema.json

```diff
@@ -0,0 +1,10 @@
+{
+    "type": "string",
```

Comment:
how about we also add `pattern` field, like `^https:\/\/(?:www\.)?linkedin\.com.*` that will validate provided link to make sure it starts with `https://www.linkedin.com/` or `https://linkedin.com/`

Reply:
yes it will work

Reply:
you used a wrong field

https://json-schema.org/understanding-json-schema/reference/regular_expressions